Library/Formula/duplicity.rb

```diff
@@ -1,8 +1,7 @@
 class Duplicity < Formula
   homepage "http://www.nongnu.org/duplicity/"
-  url "http://code.launchpad.net/duplicity/0.6-series/0.6.25/+download/duplicity-0.6.25.tar.gz"
-  sha1 "fe0b6b0b0dc7dbc02598d96567954b48c4308420"
-  revision 1
+  url "http://code.launchpad.net/duplicity/0.7-series/0.7.02/+download/duplicity-0.7.02.tar.gz"
```

Comment:
Na. If it works on 443, we use it. It usually escapes because people typically bump the URL without checking the ` http ` vs ` https ` protocol, which is understandable. The ` audit ` checks a load of this stuff these days, but can't catch everything alas. Thanks for fixing it!